jax/lax/lax.py

```diff
@@ -192,6 +192,14 @@ def atan2(x, y):
     :math:`\mathrm{atan}({x \over y})`."""
   return atan2_p.bind(x, y)
 
+def betainc(a, b, x):
+  r"""Elementwise regularized incomplete beta integral."""
+  # TODO(srvasude): Remove these once XLA broadcasts these inputs.
+  a = _brcast(_brcast(a, b), x)
```

Comment:
As a general rule, `lax` primitives don't have broadcasting semantics themselves (except for scalars). There are a few exceptions. We tend to try to expose XLA fairly directly. It would be more common to support broadcasting in the `jax.scipy.special` wrapper, if that's what scipy itself does.